.sauce/cypress.yml

```diff
@@ -29,7 +29,8 @@ suites:
   - name: "saucy test"
     browser: "chrome"
     platformName: "Windows 10"
-    screenResolution: "2560x1600"  # Available resolutions on Windows: '800x600', '1024x768', '1152x864', '1280x768', '1280x800', '1280x960', '1280x1024', '1400x1050', '1440x900', '1600x1200', '1680x1050', '1920x1080', '1920x1200', '2560x1600'
+      #screenResolution: "1920x1080"  # Available resolutions on Windows: '800x600', '1024x768', '1152x864', '1280x768', '1280x800', '1280x960', '1280x1024', '1400x1050', '1440x900', '1600x1200', '1680x1050', '1920x1080', '1920x1200', '2560x1600'
+      # If no screenResolution set, it will use 1024x768 by default
```

Comment:
It may be better above the value, to keep it consistent with here for example: https://github.com/saucelabs/saucectl/blob/master/.sauce/cypress.yml#L19

Reply:
I'll update the suggested resolution to `1920x1080` both in cypress and playwright config. And I'll update documentation accordingly.